deployment/monocular/templates/ingress.yaml

```diff
@@ -25,6 +25,10 @@ spec:
           serviceName: {{ template "fullname" $ }}-api
```

Comment:
Maybe add a note saying that this backend is deprecated?

Reply:
Perhaps not really necessary since this is going to be removed in the next couple of PRs, and this is on the dev-v1 branch so it won't affect any current users. WDYT?

Reply:
You are right, but not always assume that you should only be the one that has the context of what's next. Adding that comment, even if temporary could help others to continue your work and for sure to not to forget.